packages/flutter/lib/src/painting/box_border.dart

```diff
@@ -489,6 +503,22 @@ class Border extends BoxBorder {
         && right.strokeAlign == topStrokeAlign;
   }
 
+  Set<Color> get _distinctVisibleColors => <Color?>{
+        if (top.style == BorderStyle.none) null else top.color,
+        if (right.style == BorderStyle.none) null else right.color,
+        if (bottom.style == BorderStyle.none) null else bottom.color,
+        if (left.style == BorderStyle.none) null else left.color,
+      }.whereNotNull().toSet();
+
+  /// [BoxBorder.paintNonUniformBorder] is about 20% faster than [paintBorder],
+  /// but [paintBorder] is able to draw hairline borders when width is zero
+  /// and style is [BorderStyle.solid].
```

Comment:
these should be `//` not `///` since the text as-is wouldn't be suitable for documentation

Reply:
Ok. Done.